frontend/src/composables/useDrawer.ts

```diff
@@ -77,6 +84,17 @@ export const isItemOnTop = (
 			(item.address && item.address === currentTopItem.value.address) ||
 			(item.id && item.id === currentTopItem.value.id)
 		)
+	if (
+		item.entityTypeName === 'contract' &&
+		item.entityTypeName === currentTopItem.value.entityTypeName
+	)
+		return !!(
+			item.contractAddressIndex &&
+			item.contractAddressSubIndex &&
```

Comment:
I would assume that this check is never true as `subindex === 0` for all existing contracts. Have you tried this out?

Reply:
Yes the drawer works - I must admit I have tried follow all other cases where we have details panels and I haven’t tested if this exact block of code is actually evaluated.

I have changed to evaluate null and undefined.